pennylane/transforms/__init__.py

```diff
@@ -112,6 +112,7 @@
 .. autosummary::
     :toctree: api
 
+    ~transforms.qcut.contract_tensors
```

Comment:
```suggestion
```
Thanks for spotting @antalszava! Though we did intend to not mention this function.